M/MPICH/build_tarballs.jl

```diff
@@ -27,13 +27,14 @@ fi
     --enable-shared=yes \
     --enable-static=no \
     --disable-dependency-tracking \
-    --disable-fortran \
     --docdir=/tmp \
     "${EXTRA_FLAGS[@]}"
 
 # Build the library
 make -j${nproc}
 
+make check
```

Comment:
I don't know this specic case, but usually `make check` would run the built program, we can't do that in this environment

Reply:
You mean we can't check that the libraries, etc., link correctly against test programs? Isn't that desirable though?

Reply:
Would be great if we could run foreign programs, but we just can't.  So running `make check` is not an option if it would run programs